exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -58,7 +58,7 @@ class CesiumDataPrim "CesiumDataPrim" (
         doc = "Debug option that renders tilesets with textures disabled."
     )
 
-    bool cesium:debug:disableGeometryPool = true (
+    bool cesium:debug:disableGeometryPool = false (
```

Comment:
This isn't relate to the upgrade. Is there another PR to merge first?

Reply:
Geometry pooling doesn't work in Kit 104.2 because we can't change a prim's texcoords dynamically. We've wanted to turn it on for a while, and now we can with Kit 105.